docs/theory/enzyme_kinetics.rst

```diff
@@ -3,3 +3,119 @@ Enzyme kinetics
 ===============
 
 This document explains the assumptions about enzyme kinetics that Maud uses.
+
+Modular rate law
+================
+
+This section outlines the assumptions made with the modular rate law, and includes a derivation of 2 substrate (A, B),
+2 product (P, Q) random mechanism with competitive inhibitor I. It also highlights the general structure of the
+modular rate law used in Maud. The modular rate law framework was taken from [1],
+and was adapted to suit our structure.
+
+Assumptions
+-----------
+
+The assumptions used in the modular rate law are listed below:
+    - the metabolite binding occurs in a random order,
+    - binding does not occur simultanesouly,
+    - substrates and products cannot bind at the same time,
+    - metabolite binding rates are much higher than the interconversion of substrate to product (rapid equilibrium assumption),
+    - metabolite binding affinity is independent of order.
+
+
```

Comment:
I think it could do with another section describing the modular rate law in general terms before going into specific examples.